changelog/fix_invalid_squish_sql_heredoc.md

```diff
@@ -0,0 +1 @@
+* [#929](https://github.com/rubocop/rubocop-rails/issues/929): Prevent `Rails/SquishedSQLHeredocs` applying when single-line comments are present ([@john-h-k][])
```

Comment:
CI has failure. It seem to have forgotten the period at the end of the sentence.

```suggestion
* [#929](https://github.com/rubocop/rubocop-rails/issues/929): Prevent `Rails/SquishedSQLHeredocs` applying when single-line comments are present. ([@john-h-k][])
```